diff --git a/node_modules/react-native/Libraries/Components/TV/TVFocusGuideView.js b/node_modules/react-native/Libraries/Components/TV/TVFocusGuideView.js
index 439de05..5f4f4d0 100644
--- a/node_modules/react-native/Libraries/Components/TV/TVFocusGuideView.js
+++ b/node_modules/react-native/Libraries/Components/TV/TVFocusGuideView.js
@@ -104,10 +104,12 @@ function TVFocusGuideView({
   const mergedRef = useMergeRefs(setLocalRef, ref);
 
   React.useEffect(() => {
-    if (destinationsProp !== null && destinationsProp !== undefined) {
+    if (focusable === false) {
+      setDestinations([]);
+    } else if (destinationsProp !== null && destinationsProp !== undefined) {
       setDestinations(destinationsProp); // $FlowFixMe[incompatible-call]
     }
-  }, [setDestinations, destinationsProp]);
+  }, [setDestinations, destinationsProp, focusable]);
 
   const enabledStyle = {display: enabled ? 'flex' : 'none'};
   const style = [styles.container, props.style, enabledStyle];
@@ -124,7 +126,7 @@ function TVFocusGuideView({
       style={style}
       ref={mergedRef}
       collapsable={false}
-      autoFocus={autoFocus}
+      autoFocus={focusable === false ? true : autoFocus}
       // tvOS only prop
       isTVSelectable={tvOSSelectable}
       // Android TV only prop
